corehq/pillows/case.py

```diff
@@ -78,6 +87,7 @@ def get_case_pillow(
     processor_chunk_size=DEFAULT_PROCESSOR_CHUNK_SIZE,
     topics=None,
     dedicated_migration_process=False,
+    include_demo_topic=False,
```

Comment:
Where is this used?

Reply:
Decided to move it to a setting instead: 10b64fdfdd4fe5f3980800a23c0c6245c22745cb